### PR TITLE
Update doc to include sudo and chmod for commands

### PR DESCRIPTION
**Pull request by lagalbra**

### CODE REVIEW

**Comment by lagalbra** [comment]
@Microsoft/omsagent-devs

File: docs/OMS-MySQL-Instructions.md [hunk 1]
```diff
@@ -8,7 +8,8 @@
 
 4. Verify and update the MySQL log file path in the configuration file `/etc/opt/microsoft/omsagent/conf/omsagent.d/mysql.conf`  
 If `mysql.conf` is not present in the above location, move it:  
-`cp /etc/opt/microsoft/omsagent/sysconf/omsagent.d/mysql.conf /etc/opt/microsoft/omsagent/conf/omsagent.d/`
+`sudo cp /etc/opt/microsoft/omsagent/sysconf/omsagent.d/mysql.conf /etc/opt/microsoft/omsagent/conf/omsagent.d/`  
+`sudo chown omsagent:omsagent /etc/opt/microsoft/omsagent/conf/omsagent.d/mysql.conf`
```
Comment (robbiezhang) [inline docs/OMS-MySQL-Instructions.md]: omsagent:omiusers

**Comment by NarineM** [comment]
Sign off.

**Comment by agup006** [comment]
Sign off

File: docs/OMS-MySQL-Instructions.md (hunk 1 above)
Comment (lagalbra) [inline docs/OMS-MySQL-Instructions.md]: Done

**Comment by lagalbra** [comment]
Merged to master branch

**Pull request closed without merging**